packages/aws-rfdk/lib/deadline/test/render-queue.test.ts

```diff
@@ -1904,6 +2014,42 @@ describe('RenderQueue', () => {
       });
     });
 
+    test('not specified with TLS', () => {
```

Comment:
It took me a moment to figure out the semantics of the test name here. Perhaps a well-placed comma would help and change to `not specified, with TLS`

Reply:
Updated